tests/model_registry/conftest.py

```diff
@@ -68,7 +68,7 @@ def model_registry_db_service(
         },
         label=DEFAULT_LABEL_DICT_DB,
         annotations={
-            "template.openshift.io/expose-uri": "mysql://{.spec.clusterIP}:{.spec.ports[?(.name==\mysql\)].port}",
+            "template.openshift.io/expose-uri": r"mysql://{.spec.clusterIP}:{.spec.ports[?(.name==\mysql\)].port}",
```

Comment:
why are you making this a raw string? does the selector `.name==\mysql\` still work like this?

Reply:
yes, tested with and without